Makefile

```diff
@@ -35,7 +35,7 @@ run:
 
 test: xcbeautify
 	set -o pipefail \
-	&& swift test --disable-automatic-resolution --sanitize=thread --no-parallel \
```

Comment:
We have to drop TSan again, it keeps triggering when tests are run in parallel. It doesn't seem to be in code directly, it's in NIO. I don't think it's misuse on our end and I can't imagine it's a NIO problem, so perhaps a false positive.